openstack/cronus/templates/nebula/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.nebula.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Didn't we agree that we'll use statefulset?

Reply:
```
{{- if .Values.nebula.enabled }}
apiVersion: apps/v1
{{ if .Values.nebula.statefulSet.enabled }}
kind: StatefulSet
{{ else }}
kind: Deployment
{{ end -}}
metadata:
```